### DISCUSSION TITLE
Don't wrap an image with <figure> if no <figcaption> is present

### DISCUSSION THREAD

**Discussion by brucelawson** [craftcms/redactor#400] [Ideas]
### Description

I'm helping a client make their sites more accessible to people with diabilities. redactor is wrapping images a `<figure>` element. `<figure>` is only useful when it associates an image with a `<figcaption>`. A `<figure>` without a `<figcaption>` makes no sense. It's confusing for screenreader users (the screenreader will announce "figure", then nothing).

**Comment by andris-sevcenko** [comment]
I'm afraid the Redactor editor itself does make some opinionated decisions about such things. The markup for image tags isn't anything that we're able to influence in a meaningful way.
